postgres/datadog_checks/postgres/postgres.py

```diff
@@ -927,7 +927,7 @@ def _get_custom_queries(self, db, tags, custom_queries, programming_error):
                         continue
 
                     metric_info = []
-                    query_tags = custom_query.get('tags', [])
+                    query_tags = list(custom_query.get('tags', []))
                     query_tags.extend(tags)
```

Comment:
(nit) Could be:

```
                    query_tags = custom_query.get('tags', []) + tags
```

Reply:
does that mean we can remove line 931?

Reply:
tried `query_tags = custom_query.get('tags', []) + tags` but it doubled the instance tags

Screenshot: https://cl.ly/8ad1a10eb140